src/test/credentials/provider/ssoAccessTokenProvider.test.ts

```diff
@@ -158,34 +165,37 @@ describe('SsoAccessTokenProvider', () => {
             setUpStubCache(validRegistation)
             sandbox.stub(sut, 'authorizeClient').resolves(validAuthorization)
 
+            let errToThrow = new Error() as SDK.AWSError
+            errToThrow.code = 'ErrorNotSlowDownAuthPendingOrTimeOut'
```

Comment:
Nit: use `const`. Any reason you can't just use a `new SDK.AWSError()`? Same comments elsewhere.

Reply:
Removed.  Changing the value isn't necessary.